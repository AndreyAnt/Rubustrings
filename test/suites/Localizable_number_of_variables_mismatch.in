"Original string with one variable %@" = "Tranlation without variables";
