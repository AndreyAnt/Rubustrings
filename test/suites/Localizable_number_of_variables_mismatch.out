[34mProcessing file: "test/suites/Localizable_number_of_variables_mismatch.in"[0m
test/suites/Localizable_number_of_variables_mismatch.in:1: error: variables mismatch: "Original string with one variable %@" = "Tranlation without variables";
[31m[1m
Result: âœ˜ Some errors detected[0m[0m
